frontend/src/metabase/api/tags/utils.ts

```diff
@@ -364,9 +365,14 @@ export function providePermissionsGroupListTags(
 }
 
 export function providePermissionsGroupTags(
-  group: GroupListQuery,
+  group: Group | GroupListQuery,
 ): TagDescription<TagType>[] {
-  return [idTag("permissions-group", group.id)];
+  return [
```

Comment:
We want to refresh permissions group when a user in that group is e.g. deleted

Reply:
This could/should be refactored so there is no union and `in`

Reply:
e3bb35ce68c3555b96784c6965c1be1907f9e805

Reply:
lgtm